backend/database.js

```diff
@@ -1,12 +1,12 @@
 'use strict'
 
-import sbp from '~/shared/sbp.js'
+import sbp from '@sbp/sbp'
 import { strToB64 } from '~/shared/functions.js'
 import { Readable } from 'stream'
 import fs from 'fs'
 import util from 'util'
 import path from 'path'
-import '~/shared/domains/okTurtles/data.js'
+import '@sbp/okturtles.data'
```

Comment:
After line 125 in this file, we should also lock the selectors after overwriting them so that they cannot be overwritten again:

```js
sbp('sbp/selectors/lock', ['gi.db/get', 'gi.db/set', 'gi.db/delete'])
```